lazyflow/classifiers/tiktorchLazyflowClassifier.py

```diff
@@ -60,44 +56,60 @@ def __init__(
         binary_state: bytes,
         binary_optimizer_state: bytes,
         server_config: dict,
-        start_server: bool = True
+        start_server: bool = True,
     ) -> None:
+        # default values for config:
+        self._config = {
+            # Similar to a halo, the shrinkage describes the loss of image size on one side for each dimension
+            "shrinkage": (0, 0, 0, 0),
+            "name": "tiktorch model",
+        }
+        self._config.update(config)
+        # assert all(key not in self._config for key in self.__dict__.keys())
+        # self.__dict__.update(self._config)
+        # print('shrinkage:', self.shrinkage)
 
         # Privates
-        self._tikTorchClient = None
         self._tikTorchClassifier = None
         self._train_model = None
-        self._opReorderAxes = OpReorderAxes(graph=Graph())
-        self._opReorderAxes.AxisOrder.setValue('zcyx')
+        self._opReorderAxesIn = OpReorderAxes(graph=Graph())
+        self._opReorderAxesIn.AxisOrder.setValue("czyx")
+        self._opReorderAxesOut = OpReorderAxes(graph=Graph())
+        self._valid_shapes = []
 
-        addr, port = socket.gethostbyname(server_config['address']), server_config['port']
+        addr, port = socket.gethostbyname(server_config["address"]), server_config["port"]
         conn_conf = TCPConnConf(addr, port)
 
-        if addr == '127.0.0.1':
-            laucher = LocalServerLauncher(conn_conf)
+        if addr == "127.0.0.1":
+            self.launcher = LocalServerLauncher(conn_conf)
         else:
-            laucher = RemoteSSHServerLauncher(
-                conn_conf,
-                cred=SSHCred(server_config['username'], server_config['password'])
+            self.launcher = RemoteSSHServerLauncher(
+                conn_conf, cred=SSHCred(server_config["username"], server_config["password"])
             )
 
-        laucher.start()
+        self.launcher.start()
+        self._shutdown_sent = False
```

Comment:
Is it possible to ask Launchers about their status and remove the `_shutdown_sent` variable?